org.caveman250.gtkeddit.json

```diff
@@ -0,0 +1,163 @@
+{
+  "app-id": "org.caveman250.gtkeddit",
```

Comment:
Is it an actual domain?

Reply:
Ah no sorry, i'm quite new to all this. do i need to have a domain registered for the app? can i use a name that isnt a domain?

Reply:
No worries! It's not a requirement per se but we want reverse DNS notation used by flatpak to be connected to something you "own". It can be a domain, but also something along `io.gitlab.caveman250.gtkeddit` is also fine as it clearly shows ownership.

Reply:
thanks for that, name has been changed to io.gitlab.caveman250.gtkeddit